### PR TITLE
Create zpools / zfs datasets like ubiquity does

### PR DESCRIPTION
**Pull request by dbungert**

### CODE REVIEW

File: subiquity/models/filesystem.py [hunk 1]
```diff
@@ -1115,6 +1115,8 @@ class ZPool:
     # default dataset options for the zfses in the pool
     fs_properties: Optional[dict] = None
 
+    default_features: Optional[bool] = True
```
Comment (mwhudson) [inline subiquity/models/filesystem.py:1118]: Er this doesn't seem to match the change to made to curtin in the end. Did you forget to push an update to this branch?

Comment (dbungert) [reply subiquity/models/filesystem.py:1118]: No, I think this is correct, just confusing (and this is the less confusing version!)
See also the matching curtin side, https://git.launchpad.net/curtin/commit/?id=4195867e55953afd8c2481f813db3cda17791ead

in the simplest `zpool create $poolname $device` case, no `-d` arg is present and we include the default features.
If we want maximum control, we can pass `-d`, default_features=False, and then supply feature flags that we want.

Comment (mwhudson) [reply subiquity/models/filesystem.py:1118]: oh ok

**Review by mwhudson — APPROVED** [review]
I have not checked the consistency with ubiquity line by line. I can if you want!

**Pull request merged**